tools/acrn-crashlog/acrnprobe/conf.rst

```diff
@@ -0,0 +1,379 @@
+  + /.../dir/string[0] --> the first file of files, sorted by ``alphasort(3)``,
+    with prefix "string" under dir.
+  + /.../dir/string[-1] --> the last file of files, sorted by ``alphasort(3)``,
+    with prefix "string" under dir.
+
+Vm
+==
+
+Example:
+
+.. code-block:: xml
+
+   <vm id="1" enable="true">
+           <name>VM1</name>
+           <channel>polling</channel>
+           <interval>60</interval>
+           <syncevent id="1">CRASH/TOMBSTONE</syncevent>
+           <syncevent id="2">CRASH/UIWDT</syncevent>
+           <syncevent id="3">CRASH/IPANIC</syncevent>
+           <syncevent id="4">REBOOT</syncevent>
+   </vm>
+
+* ``name``:
+  The name of virtual machine.
+* ``channel``:
+  The ``channel`` name to get the virtual machine events.
+* ``interval``:
+  Time interval in seconds of polling vm's image.
+* ``syncevent``:
+  Event type ``acrnprobe`` will synchronize from virtual machine's ``crashlog``.
+  User could specify different types by id. The event type can also be
+  indicated by ``type/subtype``.
+
+Log
+===
+
+Example:
+
+.. code-block:: xml
+
+   <log id="1" enable="true">
+           <name>pstore</name>
+           <type>node</type>
+           <path>/sys/fs/pstore/console-ramoops-0</path>
+   </log>
+
+* ``name``:
+  By default, ``acrnprobe`` will take this ``name`` as generated log's name in
+  ``outdir`` of sender crashlog.
+  If ``path`` is specified by simple formats (includes [*], [0] or [-1]) the
+  file name of generated logs will be the same as original. More details about
+  simple formats, see `Trigger`_.
+* ``type`` and ``path``:
+  Same as `Trigger`_.
+* ``lines``:
+  By default, all contents in the original will be copied to generated log.
+  If this label is configured, only the ``lines`` at the end in the original
+  will be copied to the generated log. It takes effect only when the ``type`` is
+  ``file``.
+
+Crash
+=====
+
+Example:
+
+.. code-block:: xml
+
+   <crash id='1' inherit='0' enable='true'>
+           <name>UNKNOWN</name>
+           <trigger>t_rebootreason</trigger>
+           <channel>oneshot</channel>
+           <content id='1'>WARM</content>
+           <log id='1'>pstore</log>
+           <log id='2'>acrnlog_last</log>
+   </crash>
+   <crash id='2' inherit='1' enable='true'>
+           <name>IPANIC</name>
+           <trigger>t_pstore</trigger>
+           <content id='1'> </content>
+           <mightcontent expression='1' id='1'>Kernel panic - not syncing:</mightcontent>
+           <mightcontent expression='1' id='2'>BUG: unable to handle kernel</mightcontent>
+           <data id='1'>kernel BUG at</data>
+           <data id='2'>EIP is at</data>
+           <data id='3'>Comm:</data>
+   </crash>
+
+* ``name``:
+  The type of the ``crash``.
+* ``trigger``:
+  The trigger name of the crash.
+* ``channel``:
+  The name of channel crash use.
+* ``content`` and ``mightcontent``:
+  They're used to match crash type. The match is successful if all the
```

Comment:
Oh.  Now we're learning more about what makes a successful match.  This should be explained earlier.

Reply:
Yes, I will link this part to ``Match crash``.

Thank you for all your comments, it's really helpful to make this doc clearer and useful.
As for the comments which are not replied in the list, I will change it directly.